src/pymorize/cmorizer.py

```diff
@@ -168,12 +209,16 @@ def find_matching_rule(
             return None
         if len(matches) > 1:
             msg = f"Need only one rule to match to {data_request_variable}. Found {len(matches)}."
-            if self._pymorize_cfg.get("raise_on_multiple_rules", True):
+            if self._pymorize_cfg.get("raise_on_multiple_rules", True, parse_bool):
                 raise ValueError(msg)
             else:
                 logger.critical(msg)
                 logger.critical(
```

Comment:
Critical just appears to be "louder" in the logs, so it is easier to look for. In principle, I would suggest we stick to three (or four) levels:

1. `logger.debug` for statements that we want to suppress by default unless the user requests more verbose output
2. `logger.info` for standard print statements
3. `logger.warn`/`logger.error` for statements that are issued either when an exception is raised, or *would be* raise unless turned off by the configuration (like in the example above)

`logger.critical` could be on the level of manual print debugging, so that you can see it right away and have something to grep for.

Reply:
Thanks for the answer! Is a good guideline!